DESCRIPTION

```diff
@@ -22,6 +22,7 @@ RoxygenNote: 7.2.0
 Depends: 
     R (>= 4.1.0)
 Imports: 
+    basilisk,
```

Comment:
Can we make these new imports `Suggests` rather than `Imports`?

If we do that we will need to add checks to the function with `requireNamespace()` to fail elegantly, but I think that is probably the way we should try to go.